android/src/main/java/io/ably/lib/push/ActivationStateMachine.java

```diff
@@ -7,9 +7,6 @@
 import android.content.SharedPreferences;
 import android.support.v4.content.LocalBroadcastManager;
 
-import com.google.android.gms.tasks.OnCompleteListener;
```

Comment:
Presumably this change does not relate directly to the scope of work being undertaken under this PR. No need to change now but in future please try to make changes like this under a separate PR to make it easier for reviewers to focus on the changes required for this piece of work.

Reply:
I've just noticed that's unused so I removed it but I agree with you and will do it in new PR next time.